kuzzleio/kuzzle-admin-console
# KZL-1041-add-refresh-button-on-document-view

## What does this PR do ?
Add refresh button to document view

### How should this be manually tested?
  - Step 1 : npm run dev
  - Step 2 : Look at a document
  - Step 3 : Change it in another admin-console
  - Step 4 : Click on refresh button to see it updated

### Other changes
/
### Boyscout
/
### Screenshots (if appropriate)
/
